src/classes/RD2_TestRecurringDonationBuilder.cls

```diff
@@ -31,7 +31,7 @@
 * @author Salesforce.org
 * @date 2019
 * @group Recurring Donations
-* @description Recurring Donations Builder
+* @description Recurring Donations Builder for unit tests
```

Comment:
Somewhere I missed where this class was added, but I have the same concern about using JSON serialization to build the object